### DISCUSSION TITLE
Live stream lag time

### DISCUSSION THREAD

**Discussion by Danimix** [AzuraCast/AzuraCast#6442] [Q&A]
in live stream there is a lag time of 40 seconds, from time of connecting to server to public/listener, is there anyway to reduce this lag/delay

**Comment by racheev** [comment]
20-40!!!!! 

How to eliminate the delay in live broadcasts????  
It's the 21st century outside!   
We were streaming video through RED5, and the delay was 0.5 seconds!!!

**Reply by BusterNeece** [reply]
@racheev Calm down and be respectful.

**Reply by racheev** [reply]
Answer the question. Everyone here is calm.
